org-code-javabuilder/lib/src/main/java/org/code/javabuilder/LambdaRequestHandler.java

```diff
@@ -195,9 +187,44 @@ public String handleRequest(Map<String, String> lambdaInput, Context context) {
         System.exit(LOW_DISK_SPACE_ERROR_CODE);
       }
     }
+
     return "done";
   }
 
+  private Thread createTimeoutThread(
```

Comment:
(perhaps out of scope) Would it make sense to add this to the common code path (i.e. the code path that's shared between LambdaRequestHandler, WebSocketServer, and the local execution)? That way this gets exercised both in local development as well as with a deployed version of Javabuilder.

Reply:
I think to some extent yes - currently this thread uses lambda specific objects and values (Context, timeout values, etc) but we may be able to refactor and pull common logic into CodeExecutionManager. I'll add a note about this to the TODO above that mentions moving common steps into CodeExecutionManager.